x/paychan/codec.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+package paychan;
+
+import "github.com/iov-one/weave/x/codec.proto";
+import "github.com/iov-one/weave/crypto/models.proto";
+
+// PaymentChannel holds the state of a payment channel during its lifetime.
+message PaymentChannel {
+    // Sender is the source that the founds are allocated from.
+    crypto.PublicKey sender_public_key = 1;
+    bytes recipient = 2;
```

Comment:
req: sender address is missing. sender_public_key can be different than the sender's account key. At least this is how I had understood the idea. @ethanfrey  ?